src/d3a/models/market/metadata.json

```diff
@@ -0,0 +1,1119 @@
+{
```

Comment:
This could safely be removed as it could be generated based on need from the provided ink contract.

Reply:
This is not the case, because the names of the functions convert to hexes via the metadata file